src/coredump/coredump.c

```diff
@@ -1363,6 +1615,14 @@ static int process_kernel(int argc, char* argv[]) {
                 /* OK, now we know it's not the journal, hence we can make use of it now. */
                 log_set_target_and_open(LOG_TARGET_JOURNAL_OR_KMSG);
 
+        if (!in_same_namespace(getpid_cached(), context.pid, NAMESPACE_PID)) {
+                /* If this fails, fallback to the old behavior so that
+                 * there is still some record of the crash. */
+                r = forward_coredump_to_container(&context);
+                if (r >= 0)
+                        return 0;
+        }
```

Comment:
can you move the two iovw_put_string_field() lines from further up in the function to the end of the function? now that we redirect coredumps like this it's kinda useless doing that work for nothing.